src/core/Cargo.toml

```diff
@@ -4,7 +4,7 @@ version = "0.1.0"
 authors = ["Intel Corporation"]
 
 [dependencies]
-sawtooth_sdk = { git = "https://github.com/hyperledger/sawtooth-core.git" }
+sawtooth_sdk = { git = "https://github.com/hyperledger/sawtooth-core.git", rev="4a11e20"}
```

Comment:
Why is this specific revision needed?

Reply:
There have been changes in the sdk after this revision. This is an interim workaround to prevent Jenkins build failure. Fix in [PR#48](https://github.com/askmish/sawtooth-poet2/pull/48)